ow_faults_detection/src/FaultDetector.cpp

```diff
@@ -74,8 +74,10 @@ void FaultDetector::cameraPublishFaultMessages(bool is_fault)
   owl_msgs::CameraFaultsStatus camera_faults_msg;
```

Comment:
The header doesn't seem to be set, e.g. with `setFaultsMessageHeader`

Reply:
Fixed!